pandas/tests/series/test_sorting.py

```diff
@@ -64,6 +64,13 @@ def test_sort_values(self):
         ordered = ts.sort_values(ascending=False, na_position='first')
         assert_almost_equal(expected, ordered.valid().values)
 
+        # ascending=[False] should behave the same as ascending=False
+        ordered = ts.sort_values(ascending=[False])
+        expected = np.sort(ts.valid().values)[::-1]
+        assert_almost_equal(expected, ordered.valid().values)
+        ordered = ts.sort_values(ascending=[False], na_position='first')
+        assert_almost_equal(expected, ordered.valid().values)
+
```

Comment:
can you test a couple of invalid ones that they raise (e.g. ``[], [1, 2, 3], [False, False, False], 'foobar', None``

(ascending=None) is technically invalid.